sdk/packet/src/lib.rs

```diff
@@ -52,9 +52,10 @@ bitflags! {
         const FORWARDED      = 0b0000_0010;
         const REPAIR         = 0b0000_0100;
         const SIMPLE_VOTE_TX = 0b0000_1000;
-        const TRACER_PACKET  = 0b0001_0000;
         // Previously used - this can now be re-used for something else.
-        const UNUSED = 0b0010_0000;
+        const UNUSED_0  = 0b0001_0000;
+        // Previously used - this can now be re-used for something else.
+        const UNUSED_1 = 0b0010_0000;
```

Comment:
Do we need like a cool off period before this is re-used? So that old versions don't misinterpret as tracer packet